Assignment_1.py

```diff
@@ -0,0 +1,233 @@
+# %%
+import pandas as pd
+import numpy as np
+
+from nilearn import plotting
+import nibabel as nib
+
+from scipy import signal, stats
+
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FixedLocator, FixedFormatter
+
+import warnings
+# %%
+
+def standardize_data(bold_data, chunks):
+
+    """Standardizes and detrends the data.
+    
+    Parameters
+    ----------
+    bold_data : array_like 
+        4-D array with time dimension on last index The input data
+    
+    chunks_df : array_like
+        array containing the indicies of chenks ordered by their time
+
+    Returns
+    -------
+    temp : ndarray
+        z-score of the input data.
+    
+    """
+    for chunk in chunks:
+        bold_data[:,:,:,chunk] = \
+            stats.zscore(\
+            signal.detrend(\
+                bold_data[:,:,:,chunk],type='linear'))
```

Comment:
you don't explicitly specify along which axis the scoring and detrending need to be performed. Are you sure this is correct?